openmetadata-ui/src/main/resources/ui/src/pages/TagPage/TagPage.tsx

```diff
@@ -0,0 +1,680 @@
+  const [assetCount, setAssetCount] = useState<number>(0);
+  const [tagPermissions, setTagPermissions] = useState<OperationPermission>(
+    DEFAULT_ENTITY_PERMISSION
+  );
+  const assetTabRef = useRef<AssetsTabRef>(null);
+  const [previewAsset, setPreviewAsset] =
+    useState<EntityDetailsObjectInterface>();
+  const breadcrumb: TitleBreadcrumbProps['titleLinks'] = useMemo(() => {
+    return tagItem
+      ? [
+          {
+            name: 'Classifications',
+            url: ROUTES.TAGS,
+            activeTitle: false,
+          },
+          {
+            name: tagItem.classification?.name ?? '',
+            url: tagItem.classification?.fullyQualifiedName
+              ? getClassificationDetailsPath(
+                  tagItem.classification.fullyQualifiedName
+                )
+              : '',
+            activeTitle: false,
+          },
+        ]
+      : [];
+  }, [tagItem]);
+
+  const handleAssetClick = useCallback((asset) => {
+    setPreviewAsset(asset);
+  }, []);
+
+  const { editTagsPermission, editDescriptionPermission } = useMemo(() => {
+    if (tagItem) {
+      return {
+        editTagsPermission:
+          !tagItem.disabled &&
+          (tagPermissions.EditTags || tagPermissions.EditAll) &&
+          !tagItem.deleted,
+        editDescriptionPermission:
+          !tagItem.disabled &&
+          (tagPermissions.EditDescription ||
+            tagPermissions.EditAll ||
+            tagPermissions.EditTags) &&
+          !tagItem.deleted,
+      };
+    }
+
+    return { editTagsPermission: false, editDescriptionPermission: false };
+  }, [tagPermissions, tagItem?.deleted]);
+
+  const fetchCurrentTagPermission = async () => {
+    if (!tagItem?.id) {
+      return;
+    }
+    try {
+      const response = await getEntityPermission(
+        ResourceEntity.TAG,
+        tagItem?.id
+      );
+      setTagPermissions(response);
+    } catch (error) {
+      showErrorToast(error as AxiosError);
+    }
+  };
+
+  const onDescriptionUpdate = async (updatedHTML?: string) => {
+    if (tagItem) {
+      if (tagItem.description !== updatedHTML) {
+        const updatedTableDetails = {
+          ...tagItem,
+          description: updatedHTML,
+        };
+        const jsonPatch = compare(tagItem, updatedTableDetails);
+        try {
+          const response = await patchTag(tagItem.id ?? '', jsonPatch);
+
+          setTagItem(response);
+        } catch (error) {
+          showErrorToast(error as AxiosError);
+        }
+        setIsDescriptionEditable(false);
+      } else {
+        setIsDescriptionEditable(false);
+      }
+    }
+  };
+
+  const getTagData = async () => {
+    try {
+      setIsLoading(true);
+      if (tagFqn) {
+        const response = await getTagByFqn(tagFqn, {
+          fields: TabSpecificField.DOMAIN,
+        });
+        setTagItem(response);
+      }
+    } catch (e) {
+      if (e instanceof AxiosError) {
+        showErrorToast(e);
+      }
```

Comment:
i dont think we need this condition, we can directly show error in toast. or do we have any specific requirement?

Reply:
Yeah. we can directly show that. There is no requirement as such.